Sample App/Fattmerchant-iOS-SDK-Sample/ViewController.swift

```diff
@@ -246,7 +246,7 @@ class ViewController: UIViewController, TransactionUpdateDelegate, MobileReaderC
       }
     }, error: { (error) in
       self.log(error)
-    })
+    }, args: ["readersNeeded":0])
```

Comment:
This approach seems easy enough to work with, but not if the user has to hardcode that string. We need to provide a way for the user to pass that value in without having to copy/paste the string. I like the flexibility that passing in an `args` dict offers us though. So maybe we can keep the `args` dict and figure out how we can use a variable in place of that string